src/main/java/seedu/address/ui/HomeCard.java

```diff
@@ -75,16 +75,37 @@ public class HomeCard extends UiPart<Region> {
     private TranslateTransition moveTransition;
     private TranslateTransition moveBackTransition;
     private FadeTransition fadeInTransition;
+    private DateTimeFormatter timeFormatter;
+    private DateTimeFormatter secondFormatter;
+    private DateTimeFormatter dateFormatter;
```

Comment:
I see that you have extracted these out, but as uninitialized instance level attributes. Would it be better to have them as `private static final` constants?
```suggestion
    private static final DateTimeFormatter timeFormatter = DateTimeFormatter.ofPattern("HH:mm");;
    private static final DateTimeFormatter secondFormatter = DateTimeFormatter.ofPattern(":ss");
    private static final DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("EEEE, MMMM d");
```